living_documentation_generator/utils/utils.py

```diff
@@ -62,5 +65,5 @@ def set_action_output(name: str, value: str, default_output_path: str = "default
 
 
 def set_action_failed(message: str):
-    print(f"::error::{message}")
+    logger.error("::error:: %s", message)
```

Comment:
I am not sure here if print is not required. It needs to me tested.
Please make a TODO for Integration testing.

Reply:
Todo added in commit: [d235579](https://github.com/AbsaOSS/living-doc-generator/pull/26/commits/d23557990c0a1b474d9700e23d5ee54f3e2e4806).